packages/rule-apple-touch-icons/README.md

```diff
@@ -1,21 +1,21 @@
 # Require an apple touch icon (`@sonarwhal/rule-apple-touch-icons`)
 
-`apple-touch-icons` warns against not using a single `180×180px`
-PNG image for the `apple-touch-icon`.
+`apple-touch-icons` warns against using multiple PNG images when
+a single `180×180px` PNG image for the `apple-touch-icon` is recommended.
 
 ## Why is this important?
 
-Safari for iOS supports since `iOS 1.1.3` a way for developers
-to specify the image (known as [touch icon][touch icons]) that will
-be used when the user adds the web site/app to the homescreen.
+Safari for iOS has supported a way for developers to specify an
```

Comment:
Let's move `since iOS 1.1.3` at the beginning as having it at the end sounds a bit weird.